### PR TITLE
Update pubmed_krauthammer.bib

### PR DESCRIPTION
**Pull request by kyriakosschwarz**

### CODE REVIEW

File: assets/bib/pubmed_krauthammer.bib [hunk 1]
```diff
@@ -1,3 +1,19 @@
+@Article{Allam2019,
+author={Allam, Ahmed
+and Nagy, Mate
+and Thoma, George
+and Krauthammer, Michael},
+title={Neural networks versus Logistiregression for 30 days all-cause readmission prediction},
```
Comment (lokijuhy) [inline assets/bib/pubmed_krauthammer.bib:6]: Logistiregression: typo?

File: assets/bib/pubmed_krauthammer.bib [hunk 2]
```diff
@@ -1,3 +1,19 @@
+@Article{Allam2019,
```
Comment (lokijuhy) [inline assets/bib/pubmed_krauthammer.bib:1]: `Article` needs to be lowercase in order for the entry to be recognized by bibbase. When I run this locally, the article does not display.

Comment (kyriakosschwarz) [reply assets/bib/pubmed_krauthammer.bib:1]: It shows up on https://uzh-dqbm-cmi.github.io/cmi-website/publications/

File: assets/bib/pubmed_krauthammer.bib (hunk 1 above)
Comment (kyriakosschwarz) [reply assets/bib/pubmed_krauthammer.bib:6]: I don't get it, I copy-pasted the exact same thing from here: https://www.bruot.org/ris2bib/
for a second time and now seems fixed...

File: assets/bib/pubmed_krauthammer.bib (hunk 2 above)
Comment (lokijuhy) [reply assets/bib/pubmed_krauthammer.bib:1]: Oh, running it locally doesn't actually use the local copy of the .bib file, so I bet I just checked it before the hosted file got updated in github. Never mind!

File: assets/bib/pubmed_krauthammer.bib (hunk 1 above)
Comment (lokijuhy) [reply assets/bib/pubmed_krauthammer.bib:6]: 🤷🏻‍♀️

**Pull request merged**